android/app/src/main/java/org/fossasia/openevent/adapters/SessionsListAdapter.java

```diff
@@ -236,7 +236,7 @@ public void onBindViewHolder(final SessionViewHolder holder, final int position)
                 } else {
                     Snackbar.make(holder.sessionCard, R.string.removed_bookmark, Snackbar.LENGTH_SHORT).show();
                 }
-            } else {
+            } else if (track != null) {
```

Comment:
This logic can be reduced by using `if (track == null) return;` at the start of on click listener

Reply:
done with the requested changes